src/main/java/org/jenkinsci/plugins/p4/workspace/ManualWorkspaceImpl.java

```diff
@@ -233,6 +234,12 @@ public IClient setClient(IOptionsServer connection, String user) throws Exceptio
 		}
 		iclient.setStream(streamFullName);
 
+		String streamAtChange = getSpec().getStreamAtChange();
+		if(StringUtils.isNotEmpty(streamAtChange)){
+			streamAtChange = getExpand().format(streamAtChange,true);
+			iclient.setStreamAtChange(Integer.parseInt(streamAtChange));
```

Comment:
streamAtChange could be an automatic label (e.g. String).  If we add 64bit long support to P4Java than change will be a long and not int (but that's for the future).  If using Integer.parseInt you may want to catch the Runtime NumberFormatException.